bundles/binding/org.openhab.binding.zwave/database/products.xml

```diff
@@ -2456,9 +2456,10 @@
                 <Type>101</Type>
                 <Id>0603</Id>
             </Reference>
-            <Model>ZW20R</Model>
-            <Label lang="en">20A Tamper Resistant Duplex Receptacle</Label>
-            <ConfigFile>enerwave/zw20r.xml</ConfigFile>
+            <Model>ZW20R/ZWN-RSM2</Model>
+            <Label lang="en">20A Tamper Resistant Duplex Receptacle / Smart Dual Relay Switch Module</Label>
+            <ConfigFile VersionMax="3.9">enerwave/zw20r.xml</ConfigFile>
```

Comment:
Are you sure about this 3.9? Maybe you should use a higher number as we discussed?